src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4584,21 +4632,27 @@ private SeparatedSyntaxList<ParameterSyntax> ParseParameterList(
             out SyntaxToken open,
             out SyntaxToken close,
             SyntaxKind openKind,
-            SyntaxKind closeKind)
+            SyntaxKind closeKind,
+            bool allowOptionalIdentifier)
```

Comment:
thank you for makingt his non-optional.  i think it's good for the callers to be explicit.